README.md

```diff
@@ -1,27 +1,39 @@
-# Seq.Client.Log4Net [![Build status](https://ci.appveyor.com/api/projects/status/sxw4n1a6v9o7db2i?svg=true)](https://ci.appveyor.com/project/datalust/seq-client)
+# SEQ Implementation for log4net [![Build status](https://ci.appveyor.com/api/projects/status/sxw4n1a6v9o7db2i?svg=true)](https://ci.appveyor.com/project/datalust/seq-client)
```

Comment:
Most changes in this file can now be reverted since we're back to just a single package.

Reply:
Ok, will do that